code/ai/aicode.cpp

```diff
@@ -11457,8 +11457,10 @@ void get_absolute_wing_pos(vec3d *result_pos, object *leader_objp, int wingnum,
 
 	if (wing_index == 0)
 		wing_delta = vmd_zero_vector;
-	else if ( formation_index != -1 && wing_index < MAX_SHIPS_PER_WING) 
+	else if (!Wing_formations.empty() && (formation_index >= 0) && (wing_index < MAX_SHIPS_PER_WING)) {
+		Assertion(Wing_formations.size() < formation_index, "%s wing's formation_index %d is higher than the Wing_formations size.  Something went very wrong, go tell a coder!", Wings[wingnum].name, formation_index);
```

Comment:
Er, the inequality check is backwards.  Also CI is complaining about signed vs. unsigned.
```suggestion
		Assertion(formation_index < (int)Wing_formations.size(), "%s wing's formation_index %d is higher than the Wing_formations size.  Something went very wrong, go tell a coder!", Wings[wingnum].name, formation_index);
```

Reply:
Should be fixed now.

About half the time I'm able to code I'm literally falling asleep while doing it.